README.md

```diff
@@ -1,507 +1,838 @@
-Spring to create a mock user object and pass its credentials information along
-with each request that is performed on the `mockMvc` object:
+  public UserWrapper(User user) {
+    this.user = user;
+  }
 
-```java
-package com.flatiron.spring.FlatironSpring;
+  @Override
+  public String getUsername() {
+    return user.getUsername();
+  }
 
-import org.junit.jupiter.api.Test;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.security.test.context.support.WithMockUser;
-import org.springframework.test.web.servlet.MockMvc;
+  @Override
+  public String getPassword() {
+    return user.getPassword();
+  }
 
-import static org.hamcrest.Matchers.containsString;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
-import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+  @Override
+  public Collection<? extends GrantedAuthority> getAuthorities() {
+    return List.of(() -> "read");
+  }
 
-@SpringBootTest
-@AutoConfigureMockMvc
-class HelloControllerAcceptanceTest {
+  @Override
+  public boolean isAccountNonExpired() {
+    return true;
+  }
 
-    @Autowired
-    private MockMvc mockMvc;
-
-    @WithMockUser
-    @Test
-    void shouldGreetDefault() throws Exception {
-        mockMvc.perform(get("/hello"))
-                .andDo(print())
-                .andExpect(status().isOk())
-                .andExpect(content().string(containsString("Hello Stephanie")));
-    }
+  @Override
+  public boolean isAccountNonLocked() {
+    return true;
+  }
 
-    @WithMockUser
-    @Test
-    void shouldGreetByName() throws Exception {
-        String greetingName = "Jamie";
-        mockMvc.perform(get("/hello")
-                        .param("targetName", greetingName))
-                .andDo(print())
-                .andExpect(status().isOk())
-                .andExpect(content().string(containsString("Hello " + greetingName)));
-    }
+  @Override
+  public boolean isCredentialsNonExpired() {
+    return true;
+  }
+
+  @Override
+  public boolean isEnabled() {
+    return true;
+  }
 }
 ```
 
-You should now be able to run the acceptance test and have it pass. Make the
-same changes to your integration test and it will also pass:
+Let's break down this code a little bit more:
+
+- Create a private final `User` instance that will be passed into a
+  `UserWrapper` constructor.
+- Override the `getUsername()` method by returning the user's username:
+  `return user.getUsername();`
+- Override the `getPassword()` method by returning the user's password:
+  `return `
```

Comment:
Looks like this is missing ```user.getPassword()```

Reply:
Oh goodness! You must be able to see where my brain got distracted. I'll fix this.